test/index.test.js

```diff
@@ -1,13 +1,12 @@
 'use strict'
 
 const whyIsNodeRunning = require('why-is-node-running')
-const t = require('tap')
+const { test, before } = require('node:test')
 const proxyquire = require('proxyquire')
-const test = t.test
 const Fastify = require('fastify')
 const fastifyRedis = require('..')
 
-t.beforeEach(async () => {
+before(async () => {
```

Comment:
hmm, this has to be still beforeEach to ensure that the redis instance is flushed, right?

Reply:
Thank you for catching that, @Uzlopak. You're correct that we need to maintain the 'before each test' functionality to ensure the Redis instance is flushed before each test runs.